saltstack/freeipa/salt/ipaconsistency/init.sls

```diff
@@ -0,0 +1,3 @@
+install_checkipaconsistency:
+  cmd.run:
+    - name: pip install checkipaconsistency
```

Comment:
where would be the `checkipaconsistency` installed to? Into the salt virtualenv or outside of it? Have you managed to try it out?